doc/user/content/cloud/delete-deployments.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Delete Deployments"
+description: "Delete Materialize Cloud deployments."
+aliases:
+  - destroy-deployments
+---
+
+{{< cloud-notice >}}
+
+This functionality is only available to [administrators](../administer-workspace).
+
+Deleting a deployment destroys all data and catalog items. They cannot be restored.
+
+1. Go to the [Deployments](https://cloud.materialize.com/deployments) page.
+2. Click **Delete**.
+3. Enter the confirmation phrase, then click **Yes, destroy my deployment**.
```

Comment:
The instructions here don't quite match the UI anymore. The buttons are both called "Destroy", and you have to navigate to a particular deployment detail page; they're no longer accessible from the list.

Reply:
I'll do a second pass now to catch this kind of thing (not covered in #8216). Thanks, Nikhil!